CONTRIBUTING.md

```diff
@@ -39,15 +54,17 @@ python -m venv .venv
 source .venv/bin/activate
```

Comment:
Maybe we should split the `source .venv/bin/activate` line into a new code block, and explain that the virtual environment needs to be activated each time the user wishes to use the virtual environment